pkg/security/secl/model/unmarshallers.go

```diff
@@ -16,11 +16,15 @@ import (
 
 const (
 	// EventFlagsAsync async event
-	EventFlagsAsync = 1 << 0
+	EventFlagsAsync = 1 << iota
 	// EventFlagsSavedByAD saved by ad
-	EventFlagsSavedByAD = 1 << 1
+	EventFlagsSavedByAD
 	// EventFlagsActivityDumpSample an AD sample
-	EventFlagsActivityDumpSample = 1 << 2
+	EventFlagsActivityDumpSample
+	// EventFlagsSecurityProfileFoundAndAbsent if the event don't exists in the secprofile
+	EventFlagsSecurityProfileFoundAndAbsent
```

Comment:
Sorry it's a nit again, but I got confused because I didn't understand why the security profile would be `found` and `absent` 😆 Can we get away with a simple `InProfile` flag ? (the fact that there is a profile should already be in the security profile context)